src/index.js

```diff
@@ -1,9 +1,10 @@
+/** @format */
+
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 import store from './redux/configureStore';
```

Comment:
You guys made a great effort, however, there are some lines of codes to fix before approval.

- [ ] On your Navbar, you have four links instead of three. Suppose you are two people in the team you are only required to create from `Rockets` & `Missions`. I can see three links for `Dragons` as well, leading me to the `My profile` page. Also, the design of the Navbar is slightly different from the original file. Kindly implement the changes that I have mentioned.

> YOUR DESIGN
> <img width="361" alt="Screen Shot 2022-08-26 at 06 22 48" src="https://user-images.githubusercontent.com/69011963/186796466-45a33b72-7443-4241-a481-99ffc70fbb0b.png">
> 
> ORIGINAL DESIGN
> <img width="361" alt="Screen Shot 2022-08-26 at 06 09 52" src="https://user-images.githubusercontent.com/69011963/186796318-d499fb8a-c0b9-40d6-8207-8056d53caca5.png">
>